packages/utils/src/utils/agent.utils.ts

```diff
@@ -42,3 +42,102 @@ export const createAgent = async ({
     shouldFetchRootKey: fetchRootKey,
   });
 };
+
+export interface AgentManagerConfig {
```

Comment:
In another PR, we should extract an interface for all the paramters of `@dfinity/utils.createAgent` and this interface become a subtype of this new interface.

```
export type AgentManagerConfig = Omit<NewInterface, 'identity'>
```

Reply:
ah! right! shall i do it before this one? i think it would be a fast one

Reply:
Sure, go for it

Reply:
Done with:

- PR #735

Reply:
done, @peterpeterparker 

ready for another review